numpy/core/fromnumeric.py

```diff
@@ -3562,8 +3562,8 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue):
 
     See Also
     --------
-    std, mean, nanmean, nanstd, nanvar
-    ufuncs-output-type
+    numpy.std, numpy.mean, numpy.nanmean, numpy.nanstd, numpy.nanvar
+    numpy.ufuncs-output-type
```

Comment:
Note that the `ufuncs-output-type` is a label defined in a document, so this doesn't work to resolve the link here.